src/_pytest/terminal.py

```diff
@@ -597,6 +598,16 @@ def pytest_report_header(self, config):
         plugininfo = config.pluginmanager.list_plugin_distinfo()
         if plugininfo:
             result.append("plugins: %s" % ", ".join(_plugin_nameversions(plugininfo)))
+
+        if config._implicit_args:
+            result.append(
```

Comment:
I suggest we also introduce a new option, `show_implicit_arguments` defaulting to True, so users can turn this off in case they use secrets in their options or their output is too cluttered.